commit d2059d1237e23cad278af3eb717cc6e95f187587
Author: Tomasz Figa <tomasz.figa@gmail.com>
Date:   Tue Sep 7 18:44:00 2010 +0200

    Changed reserved memory and PMEM configs to enable PMEM for GPU1.

diff --git a/arch/arm/plat-s3c64xx/devs.c b/arch/arm/plat-s3c64xx/devs.c
index 74c15c2..5421e86 100755
--- a/arch/arm/plat-s3c64xx/devs.c
+++ b/arch/arm/plat-s3c64xx/devs.c
@@ -622,7 +622,7 @@ static struct android_pmem_platform_data pmem_pdata = {
  
 static struct android_pmem_platform_data pmem_gpu1_pdata = {
 	.name		= "pmem_gpu1",
-	.no_allocator	= 1,
+//	.no_allocator	= 1,
 	.cached		= 1,
 	.buffered	= 1,
 };
diff --git a/arch/arm/plat-s3c64xx/include/plat/reserved_mem.h b/arch/arm/plat-s3c64xx/include/plat/reserved_mem.h
index 435bfe5..953dfa0 100755
--- a/arch/arm/plat-s3c64xx/include/plat/reserved_mem.h
+++ b/arch/arm/plat-s3c64xx/include/plat/reserved_mem.h
@@ -43,7 +43,8 @@ extern void sec_log_buf_init(void);
 #define RESERVED_PMEM_RENDER	  	(2 * 1024 * 1024)
 #define RESERVED_PMEM_STREAM	  	(2 * 1024 * 1024)
 #define RESERVED_G3D			(32 * 1024 * 1024) 	/* G3D is shared with uppper memory areas */
-#define RESERVED_PMEM_GPU1		(0)
+//#define RESERVED_PMEM_GPU1		(0)
+#define RESERVED_PMEM_GPU1		(RESERVED_G3D)
 #define RESERVED_PMEM			(8 * 1024 * 1024)
 #define RESERVED_PMEM_SKIA		(0)
 
@@ -60,7 +61,8 @@ extern void sec_log_buf_init(void);
 #define RENDER_RESERVED_PMEM_START	(PREVIEW_RESERVED_PMEM_START - RESERVED_PMEM_RENDER)
 #define STREAM_RESERVED_PMEM_START	(RENDER_RESERVED_PMEM_START - RESERVED_PMEM_STREAM)
 #define G3D_RESERVED_START		(RESERVED_PMEM_END_ADDR - RESERVED_G3D)		 /* G3D is shared */
-#define GPU1_RESERVED_PMEM_START	(G3D_RESERVED_START - RESERVED_PMEM_GPU1)
+//#define GPU1_RESERVED_PMEM_START	(G3D_RESERVED_START - RESERVED_PMEM_GPU1)
+#define GPU1_RESERVED_PMEM_START	(G3D_RESERVED_START)
 #define RESERVED_PMEM_START		(GPU1_RESERVED_PMEM_START - RESERVED_PMEM)
 #define PHYS_UNRESERVED_SIZE		(RESERVED_PMEM_START - PHYS_OFFSET)
 
